src/foundry/foundry.js/hooks.d.ts

```diff
@@ -162,6 +162,22 @@ declare global {
         change: EffectChangeData
       ) => boolean | void;
 
+      /**
+       * A hook event that fires immediately prior to PIXI Application construction with the configuration parameters.
+       * @param canvasConfig - Canvas configuration parameters that will be used
+       */
+      canvasConfig: (canvasConfig: {
+        view: HTMLCanvasElement;
+        width: number;
+        height: number;
+        transparent: boolean;
+        resolution: number;
+        autoDensity: boolean;
+        backgroundColor: string | null;
+        antialias: boolean;
+        powerPreference: string;
+      }) => unknown;
```

Comment:
I know it's being done like that for other hooks, too, but why are we using `undefined` instead of `void`?

Reply:
I have no idea anymore honestly. Might be because typescript likely complains if you return anything with `void`, whereas it shouldn't with `unknown`.

Reply:
No, it will work fine. you can pass a function that returns anything as a parameter that is expected to return void. I think it's ok to just go ahead with this now and clean it up as a follow-up. Maybe create an issue for that.